src/components/TabSelector/TabSelector.tsx

```diff
@@ -121,7 +112,7 @@ function TabSelector({state, navigation, onTabPress, position}) {
 
                     if (!event.defaultPrevented) {
                         // The `merge: true` option makes sure that the params inside the tab screen are preserved
-                        navigation.navigate({name: route.name, merge: true});
+                        navigation.navigate({key: route.key, merge: true});
```

Comment:
Could you clarify why this was changed?

Reply:
The previous usage wasn't interpreted by typescript properly, I would have to use 
`navigation.navigate({name: route.name, merge: true, params: {});` because `params` is not optional in navigate function arguments. Thats why I switched to `navigation.navigate({key: route.key, merge: true});` usage, because it does the same thing, but has optional params argument.